src/core/src/pass/serialize.cpp

```diff
@@ -8,11 +8,11 @@
 #include <cassert>
 #include <cstdint>
 #include <fstream>
+#include <ngraph/runtime/reference/convert.hpp>
```

Comment:
why?

Reply:
for some reason arm compilation fails as convert.hpp does not even exist. The only think i could possible think of is to replace `""` wtih `<>` so that it will look by relative path instead of absolute.

>../../../../repos/openvino/src/core/src/pass/serialize.cpp:166:13: error: ���convert_from_f32_to_f16_with_clamp��� is not a member of ���ngraph::runtime::reference���
[2023-08-11T13:48:53.056Z] [2023-08-11 13:48:40,885] [140376748472064] build ie ERROR: make[2]: *** [src/core/CMakeFiles/ngraph_obj.dir/src/pass/serialize.cpp.o] Error 1

Reply:
Better to include `openvino` instead of `ngraph`, deal with it after merging into main PR.